extension/agenthealth/handler/stats/handler_test.go

```diff
@@ -37,9 +37,12 @@ func TestStatsHandler(t *testing.T) {
 		ThreadCount:          aws.Int32(789),
 		LatencyMillis:        aws.Int64(1234),
 		PayloadBytes:         aws.Int(5678),
-		StatusCode:           aws.Int(200),
```

Comment:
This wasn't removed from the structure, so I wouldn't expect to see it removed from the test.

Reply:
Oops, added this back